src/calendar-app/calendar/model/CalendarModel.ts

```diff
@@ -865,3 +1024,11 @@ export function formatNotificationForDisplay(eventTime: Date, summary: string):
 
 	return { body, title: body }
 }
+
+function loadAllEvents(groupRoot: CalendarGroupRoot): Promise<Array<CalendarEvent>> {
```

Comment:
How about running these requests in parallel?

```typescript
const [longEvents, shortEvents] = await Promise.all([
  locator.entityClient.loadAll(CalendarEventTypeRef, groupRoot.longEvents),
  locator.entityClient.loadAll(CalendarEventTypeRef, groupRoot.shortEvents)
])
return shortEvents.concat(longEvents)
```

Reply:
Yes, I forgot to update it, thanks.